### PR TITLE
fixing pr issue

### PR DESCRIPTION
**Pull request by sammy-code98**
#### PR Checklist

- [ ] `npm run lint` passes

### CODE REVIEW

File: assets/admin_icons/logout.svg
[the remark this answers is not in the document]
Comment (sammy-code98) [reply assets/admin_icons/logout.svg:1]: icons here are all displayed via an image tag

**Comment by sammy-code98** [comment]
> 1. This branch says `fix/logout`, but does a whole lot than just logout. Please let's stick to the tasks* to avoid overwhelming reviewers with too much files to go through.
> 2. I tried to login but I wasn't able to as I keep getting `401` which means my token does not get saved at all or something else I barely tried looking through.
> 3. There were a couple of `side effects` which were supposed to run once but were changed in this PR, can we go over them again to avoid unhandled effects running in
>    the _background_.
> 
> Screenshot 2024-06-04 at 1 18 30 PM

- Sincere apologies for overloading a single pr with many file changes, had a few conflicts on my end that resulted in that. will do better in the future.
-  the thing with the side effects was a huge mistake on my end, thanks for pointing them out. will fix asap
- will get other issues raised along side

File: components/molecules/LogOutModal/index.tsx
Comment (chimdie) [inline components/molecules/LogOutModal/index.tsx:1]: Add the mobile functionality for the logout

File: components/molecules/LogOutModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+              "border border-solid text-sm border-[#0D5520] px-12 py-2 rounded-lg w-1/2  md:w-auto"
+            }
+          >
+            <span className="flex gap-4 mx-auto item-center justify-center font-bold">Cancel</span>
+          </Button>
+
+          <Button
+            onClick={onLogOut}
+            classes={
+              "bg-[#0D5520] text-sm text-[white] px-12 py-2 rounded-lg w-1/2  md:w-auto"
```
Comment (chimdie) [inline components/molecules/LogOutModal/index.tsx:34]: Remove arbitrary value

**Review by chimdie — CHANGES_REQUESTED** [review]

**Review by chimdie — APPROVED** [review]
LGTM

**Pull request merged**